io.github.aganzha.Stage.json

```diff
@@ -0,0 +1,60 @@
+{
+   "app-id": "io.github.aganzha.Stage",
+   "runtime": "org.gnome.Platform",
+   "runtime-version": "46",
+   "sdk": "org.gnome.Sdk",
+   "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+   "command": "stage",
+   "finish-args": [
+      "--filesystem=home",
+      "--share=ipc",
+      "--share=network",
+      "--socket=fallback-x11",
+      "--socket=wayland",
+      "--socket=ssh-auth",
+      "--socket=gpg-agent",
+      "--device=dri"
+   ],
+   "build-options": {
+      "append-path": "/usr/lib/sdk/rust-stable/bin"
+   },
+   "modules": [{
+      "name": "stage",
+      "buildsystem": "simple",
+      "build-options": {
+         "build-args": [
+            "--share=network"
```

Comment:
Hey. The app has lots of dependencies from https://crates.io/
When building without this argument, i 've got
```warning: spurious network error (3 tries remaining): [6] Couldn't resolve host name (Could not resolve host: index.crates.io)```
How this could be avoided?

Reply:
Saw it here https://github.com/flatpak/flatpak/issues/996

Reply:
**this is not allowed** on Flathub.

https://github.com/flatpak/flatpak-builder-tools/tree/master/cargo